src/OperationalInsights/OperationalInsights/help/New-AzOperationalInsightsSearchTable.md

```diff
@@ -0,0 +1,261 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.OperationalInsights.dll-Help.xml
+Module Name: Az.OperationalInsights
+online version: https://docs.microsoft.com/powershell/module/az.operationalinsights/Create-AzOperationalInsightsSearchTable
+schema: 2.0.0
+---
+
+# New-AzOperationalInsightsSearchTable
+
+## SYNOPSIS
+Create a Search table
+
+## SYNTAX
+
+```
+New-AzOperationalInsightsSearchTable [-ResourceGroupName] <String> [-WorkspaceName] <String>
+ [-TableName] <String> [[-RetentionInDays] <Int32>] [-TotalRetentionInDays <Int32>] -SearchQuery <String>
+ -StartSearchTime <String> -EndSearchTime <String> [-Limit <Int32>] [-AsJob]
+ [-DefaultProfile <IAzureContextContainer>] [-WhatIf] [-Confirm] [<CommonParameters>]
+```
+
+## DESCRIPTION
+Create a Search table
+
+## EXAMPLES
+
+### Example 1
+```powershell
+PS C:\> New-AzOperationalInsightsSearchTable -ResourceGroupName {rgName} -WorkspaceName {wsName} -TableName {tableName_SRCH} -SearchQuery "Heartbeat"  -StartSearchTime "05-27-2022 12:26:36" -EndSearchTime "05-28-2022 12:26:36"
```

Comment:
```suggestion
New-AzOperationalInsightsSearchTable -ResourceGroupName {rgName} -WorkspaceName {wsName} -TableName {tableName_SRCH} -SearchQuery "Heartbeat"  -StartSearchTime "05-27-2022 12:26:36" -EndSearchTime "05-28-2022 12:26:36"
```
- Please remove PS C:\> from examples to avoid breaking user copy-paste-use experience.
- Please remove {} from examples to avoid our analyzer can't recognize them.
- Please split example into code and output section with following format:
\```powershell
{code}
\```
\```output
{output}
\```